doc/classes/Transform2D.xml

```diff
@@ -20,6 +20,7 @@
 			<return type="Transform2D" />
 			<description>
 				Constructs a [Transform2D] identical to [constant IDENTITY].
+				[b]Note:[/b]: In C#, constructs a zero-initialized [Transform2D] with all components set to [code]0.0[/code]. Use [constant IDENTITY] instead.
```

Comment:
I don't feel like "Use [constant IDENTITY] instead" has any particular meaning here, if not implying that base constructor is useless, which it probably is but developers do as they please.
```suggestion
				[b]Note:[/b]: In C#, this constructs a [Transform2D] with all of its components set to [code]0.0[/code].
```

Reply:
I was following the example of the existing [C# API difference page](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_differences.html#transform3d), which does specifically say to use `IDENTITY` instead. But leaving that implicit is fine too

Reply:
The API difference page says "_Please use `Transform2D.Identity` for the equivalent of `Transform2D()` in GDScript and C++._". It gives a reason for doing it, instead of "_just use the constant, trust me_". You could mention this, but it's still redundant information, I feel.

Reply:
Yeah, I left it out and applied the changes as you suggested. Better for the class ref to err on the shorter side.